security/sec-iride/src/main/java/org/geoserver/security/iride/IrideRoleService.java

```diff
@@ -95,6 +95,8 @@
     private HttpClient httpClient = new HttpClient();
     private HttpConnectionManagerParams params = new HttpConnectionManagerParams();
 
+    private String username;

```

Comment:
Instead of storing the given username in an instance field, that same information will be retrieved using: 

`SecurityContextHolder.getContext().getAuthentication().getPrincipal()`

where "principal" is going to be the username